Tools/Scripts/libraries/webkitscmpy/webkitscmpy/program/revert.py

```diff
@@ -236,17 +236,24 @@ def relate_issues(cls, args, repository, issue, commit_issues, revert_reason):
         for r_link in CommitProgram.bug_urls(issue):
             r_issue = Tracker.from_string(r_link)
             for c_issue in commit_issues.get(r_issue.tracker.NAME, []):
-                c_issue.open(why="Reopened {}.\n{}, tracking revert in {}.".format(r_issue.tracker.NAME, revert_reason, r_issue.link))
+                try:
+                    c_issue.open(why="Reopened {}.\n{}, tracking revert in {}.".format(r_issue.tracker.NAME, revert_reason, r_issue.link))
+                except radar.Tracker.radarclient().exceptions.UnsuccessfulResponseException as e:
+                    sys.stderr.write('Failed to re-open {}\n'.format(c_issue.link))
```

Comment:
We can still make a comment on a closed radar, I believe, and we should still probably do that.